benchmarks/fixed_pool_stress_test.cpp

```diff
@@ -1,122 +1,42 @@
- * then do NUM_ALLOC deallocations in the same order. The time is averaged across NUM_RND rounds. 
- */
-void same_order(umpire::Allocator alloc, int SIZE)
-{
-  double time[2] = {0.0, 0.0};
-  void* allocations[NUM_ALLOC];
-
-  for(int i = 0; i < NUM_RND; i++) {
-    auto begin_alloc = std::chrono::system_clock::now();
-    for (int j = 0; j < NUM_ALLOC; j++)
-      allocations[j] = alloc.allocate(SIZE);
-    auto end_alloc = std::chrono::system_clock::now();
-    time[0] += std::chrono::duration<double>(end_alloc - begin_alloc).count()/NUM_ALLOC;
-
-    auto begin_dealloc = std::chrono::system_clock::now();
-    for (int h = 0; h < NUM_ALLOC; h++)
-      alloc.deallocate(allocations[h]);
-    auto end_dealloc = std::chrono::system_clock::now();
-    time[1] += std::chrono::duration<double>(end_dealloc - begin_dealloc).count()/NUM_ALLOC;
-  }
-
-  alloc.release();
-  double alloc_t{(time[0]/double(NUM_RND)*CONVERT)};
-  double dealloc_t{(time[1]/double(NUM_RND)*CONVERT)};
-
-  std::cout << "  SAME_ORDER (FixedPool - " << (long long int)SIZE*FACTOR << "):" << std::endl; 
-  std::cout << "    alloc: " << alloc_t << "(us)" << std::endl;
-  std::cout << "    dealloc: " << dealloc_t << "(us)" << std::endl;
-  std::cout << "    lifetime: " << alloc_t+dealloc_t << "(us)" << std::endl << std::endl;
-}
-
-/*
- * This function measures the time it takes to do NUM_ALLOC allocations and 
- * then do NUM_ALLOC deallocations in reverse order. The time is averaged across NUM_RND rounds. 
- */
-void reverse_order(umpire::Allocator alloc, int SIZE)
-{
-  double time[2] = {0.0, 0.0};
-  void* allocations[NUM_ALLOC];
-
-  for(int i = 0; i < NUM_RND; i++) {
-    auto begin_alloc = std::chrono::system_clock::now();
-    for (int j = 0; j < NUM_ALLOC; j++)
-      allocations[j] = alloc.allocate(SIZE);
-    auto end_alloc = std::chrono::system_clock::now();
-    time[0] += std::chrono::duration<double>(end_alloc - begin_alloc).count()/NUM_ALLOC;
-
-    auto begin_dealloc = std::chrono::system_clock::now();
-    for (int h = (NUM_ALLOC-1); h >=0; h--)
-      alloc.deallocate(allocations[h]);
-    auto end_dealloc = std::chrono::system_clock::now();
-    time[1] += std::chrono::duration<double>(end_dealloc - begin_dealloc).count()/NUM_ALLOC;
-  }
-
-  alloc.release();
-  double alloc_t{(time[0]/double(NUM_RND)*CONVERT)};
-  double dealloc_t{(time[1]/double(NUM_RND)*CONVERT)};
-
-  std::cout << "  REVERSE_ORDER (FixedPool - " << (long long int)SIZE*FACTOR << "):" << std::endl; 
-  std::cout << "    alloc: " << alloc_t << "(us)" << std::endl;
-  std::cout << "    dealloc: " << dealloc_t << "(us)" << std::endl;
-  std::cout << "    lifetime: " << alloc_t+dealloc_t << "(us)" << std::endl << std::endl;
-}
+const int NUM_RND {1000}; //number of rounds (used to average timing)
+const int NUM_ALLOC {512}; //number of allocations used for testing
+const int OBJECTS_PER_BLOCK {1<<11}; //2^11 == 2048 blocks of object_bytes size
```

Comment:
Lol - Thinking about comments from @davidbeckingsale yesterday, maybe the world isn't completely ready for bit shifting :-).  Either way, I think there is a typo and suggest that this might be fixed in one of two ways:
```suggestion
const int OBJECTS_PER_BLOCK {1<<11}; //1^11 == 2048 blocks of object_bytes size
```
or
```suggestion
const int OBJECTS_PER_BLOCK {2 * 1024}; // # of blocks of object_bytes size
```

Reply:
1^11 != 2048 though lol
Maybe I'll comment "1<<11 == 2^11 == 2048 ...."  haha

Reply:
lol - you're right.  But I don't know that 1<<11 is the same as 2 exclusive or'd with 11.